src/modules/Auth/components/success/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import withTheme from 'components/shared/withTheme';
+import { PrimaryButton } from 'components/shared/toolBox/button';
+import getStyles from './styles';
+
+const SuccessScreen = ({
+  Illustration, title, description, styles, onContinue, buttonText, children
+}) => <View style={[styles.wrapper, styles.theme.wrapper]} >
+    <View style={[styles.container]} >
+      <View style={styles.illustration} >
+        <Illustration />
+      </View>
+      <Text style={[styles.title, styles.theme.title]} >{title}</Text>
+      <Text style={[styles.description, styles.theme.description]} >{description}</Text>
+      {children}
+    </View>
+    <PrimaryButton
+      title={buttonText}
+      style={styles.continueButton}
+      noTheme
+      onPress={onContinue}
+    />
```

Comment:
```suggestion
    <PrimaryButton
      noTheme
      title={buttonText}
      style={styles.continueButton}
      onPress={onContinue}
    />
```

just for consistency though